test/test_drop_duplicates.py

```diff
@@ -1,27 +0,0 @@
-# Copyright (c) 2014. Mount Sinai School of Medicine
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from pyensembl import EnsemblRelease
-from varcode import Variant, VariantCollection
-
-def test_drop_duplicates():
```

Comment:
I'm not sure about dropping this test entirely, we still probably want to ensure that the expect behavior happens when you initialize a variant collection with duplicates (though now that expected behavior is just the second assert below).